code/game/machinery/cryopod.dm

```diff
@@ -140,32 +140,19 @@ GLOBAL_LIST_EMPTY(cryopod_computers)
 	var/obj/machinery/computer/cryopod/control_computer
 	COOLDOWN_DECLARE(last_no_computer_message)
 
-	// These items are preserved when the process() despawn proc occurs.
+	// A few outlier items to be preserved on despawn_occupant()
 	var/static/list/preserve_items = list(
-		/obj/item/hand_tele,
-		/obj/item/card/id/advanced/gold/captains_spare,
-		/obj/item/aicard,
-		/obj/item/mmi,
-		/obj/item/paicard,
-		/obj/item/gun,
-		/obj/item/pinpointer,
 		/obj/item/clothing/shoes/magboots,
-		/obj/item/areaeditor/blueprints,
-		/obj/item/clothing/head/helmet/space,
-		/obj/item/clothing/suit/space,
-		/obj/item/clothing/suit/armor,
-		/obj/item/defibrillator/compact,
-		/obj/item/reagent_containers/hypospray/cmo,
 		/obj/item/clothing/accessory/medal/gold/captain,
 		/obj/item/clothing/gloves/krav_maga,
-		/obj/item/nullrod,
-		/obj/item/tank/jetpack,
-		/obj/item/documents,
-		/obj/item/nuke_core_container
+
 	)
 	// These items will NOT be preserved
 	var/static/list/do_not_preserve_items = list (
 		/obj/item/mmi/posibrain
+		/obj/item/clothing
+		/obj/item/tools
```

Comment:
What is `/obj/item/tools`? I've never seen this before.

What's the point of this list, anyway? Is this another list we'll have to maintain?

Reply:
This was nixed and yes it was

Reply:
see comment earlier on this (i'm getting absolutely buried in sub conversations so here):

>attempted to invert this list but ended up filling the do_not_preserve list just as much, ex: pdas, ids, clothing, etc. The list ended up getting so large there was basically no point (nor dynamism) to it.